File: appoointment_parent/appointment_queue/src/main/java/com/attx/service/ProjectService.java
package com.attx.service;

import java.util.List;

import com.attx.pojo.Project;

import com.attx.entity.PageResult;

public interface ProjectService {

	/**
	 * 查询列表
	 * @return
	 */
	public List<Project> getProjectList();
	
	/**
	 * 分页查询
	 * @param pageNum
	 * @param pageSize
	 * @return
	 */
	PageResult findPage(int pageNum, int pageSize);
	
	/**
	 * 修改状态
	 */
	public void updateStatus(Integer proId, String status);

	/**
	 * 设置批数上线
	 */
	public void updateAllGroup(Integer proId,String allGroup);

	/**
	 * 根据ID获取实体
	 * @param proId
	 * @return
	 */
	public Project findOne(Integer proId);

	/**
	 * 批量删除，修改isdelete字段   1正常2删除
	 */
	public void delete(Integer[] ids);

	/**
	 * 新增
	 * @param project
	 */
	public void add(Project project);
	
}

File: appoointment_parent/appointment_queue/src/main/webapp/js/service/projectService.js
//服务层
app.service('projectService',function($http) {

    //读取列表数据绑定到表单中
    this.findAll = function () {
        return $http.get("../project/getProjectList.do");
    }
    
    //分页 
	this.findPage=function(page,rows){
		return $http.get('../project/findpage.do?page='+page+'&rows='+rows);
	}

    //增加
    this.add=function(entity){
        return  $http.post('../project/add.do',entity );
    }

    // 修改状态
    this.updateStatus=function(proId,status){
        return $http.get("../project/updateStatus.do?proId="+proId+"&status="+status);
    }
    
    // 设置批数上限
    this.updateAllGroup=function(proId,status){
        return $http.get("../project/updateAllGroup.do?proId="+proId+"&allGroup="+allGroup);
    }

    //查询实体
    this.findOne=function(proId){
        return $http.get('../project/findOne.do?proId='+proId);
    }

});